### ISSUE TITLE
Errors on "cannot be imported again. It was already imported This is most likely a build authoring error. This subsequent import will be ignored"

### ISSUE THREAD

**Issue by Xarkam** [OmniSharp/omnisharp-vscode#3987]
## Issue Description ##
I receive many error on messages as :
```
Error: "/home/xarkam/.vscode/extensions/ms-dotnettools.csharp-1.23.0/.omnisharp/1.36.1/omnisharp/.msbuild/Current/Bin/Microsoft.WorkflowBuildExtensions.targets" cannot be imported again. It was already imported at "/home/xarkam/.vscode/extensions/ms-dotnettools.csharp-1.23.0/.omnisharp/1.36.1/omnisharp/.msbuild/Current/Bin/Microsoft.Common.CurrentVersion.targets (6160,3)". This is most likely a build authoring error. This subsequent import will be ignored. [/home/xarkam/Developpement/Base/src/SoftinuxBase.Infrastructure/SoftinuxBase.Infrastructure.csproj]
```
## Steps to Reproduce ##
I don't know how to reproduce it maybe it is something with my project or the latest omnisharp update
## Expected Behavior ##
Fix the errors
## Actual Behavior ##
I am notified of errors in omnisharp
### OmniSharp log ###
[onmisharp.log](https://github.com/OmniSharp/omnisharp-vscode/files/5080394/onmisharp.log)

### C# log ###
<details>Post the output from Output-->C# here</details>

## Environment information ##

**VSCode version**: 1.48.0
**C# Extension**: 1.23.0

<details><summary>Mono Information</summary>
        OmniSharp using built-in mono</details>
<details><summary>Dotnet Information</summary>
.NET Core SDK (reflecting any global.json):
 Version:   3.1.401
 Commit:    39d17847db

Runtime Environment:
 OS Name:     neon
 OS Version:  20.04
 OS Platform: Linux
 RID:         linux-x64
 Base Path:   /usr/share/dotnet/sdk/3.1.401/

Host (useful for support):
  Version: 3.1.7
  Commit:  fcfdef8d6b

.NET Core SDKs installed:
  3.1.401 [/usr/share/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.App 3.1.7 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.7 [/usr/share/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
</details>
<details><summary>Visual Studio Code Extensions</summary>

|Extension|Author|Version|
|---|---|---|
|better-comments|aaron-bond|2.1.0|
|Bookmarks|alefragnani|11.3.1|
|bracket-pair-colorizer-2|CoenraadS|0.2.0|
|csharp|ms-dotnettools|1.23.0|
|csharpextensions|jchannon|1.3.1|
|dart-code|Dart-Code|3.13.2|
|debugger-for-chrome|msjsdiag|4.12.9|
|discord-vscode|icrawl|4.0.0|
|docomment|k--kato|0.1.18|
|dotnet-test-explorer|formulahendry|0.7.4|
|emojisense|bierner|0.7.0|
|errorlens|usernamehw|3.2.0|
|flutter|Dart-Code|3.13.2|
|gistfs|vsls-contrib|0.1.8|
|git-graph|mhutchie|1.25.0|
|githistory|donjayamanne|0.6.9|
|gitignore|codezombiech|0.6.0|
|gitlens|eamodio|10.2.2|
|html-css-class-completion|Zignd|1.19.0|
|html-snippets|abusaidm|0.2.1|
|html5-boilerplate|sidthesloth|1.0.3|
|LogFileHighlighter|emilast|2.9.0|
|output-colorizer|IBM|0.1.2|
|path-intellisense|christian-kohler|2.2.1|
|polacode-with-copy|fy|0.3.4|
|prettier-vscode|esbenp|5.1.3|
|python|ms-python|2020.8.101144|
|quicktype|quicktype|12.0.46|
|remote-wsl|ms-vscode-remote|0.44.4|
|rest-client|humao|0.24.1|
|trailing-spaces|shardulm94|0.3.1|
|vetur|octref|0.26.1|
|vscode-coverage-gutters|ryanluker|2.6.0|
|vscode-eslint|dbaeumer|2.1.8|
|vscode-firefox-debug|firefox-devtools|2.9.0|
|vscode-html-css|ecmel|0.2.3|
|vscode-icons|vscode-icons-team|10.2.0|
|vscode-nuget-gallery|patcx|0.0.22|
|vscode-nuget-package-manager|jmrog|1.1.6|
|vscode-solution-explorer|fernandoescolar|0.3.10|
|vscode-typescript-tslint-plugin|ms-vscode|1.2.3|
|vscode-versionlens|Hoffs|0.25.6|
|vscode-versionlens|pflannery|1.0.8|
|vscodeintellicode|VisualStudioExptTeam|1.2.10|
|vsliveshare|ms-vsliveshare|1.0.2619|;

</details>

**Comment by RikkiGibson** [comment]
I was seeing a lot of these errors on my Mac this morning till I downgraded the extension to 1.22.1.

**Comment by Xarkam** [comment]
For information,  I don't have any error on windows. 
#### This is my logs in windows:
<details>
Starting OmniSharp server at 17/08/2020 à 17:55:40
    Target: e:\Developpement\Base\SoftinuxBase.sln

OmniSharp server started.
    Path: c:\Users\xarkam\\.vscode\extensions\ms-dotnettools.csharp-1.23.0\.omnisharp\1.36.1\OmniSharp.exe
    PID: 16580

[info]: OmniSharp.Stdio.Host
        Starting OmniSharp on Windows 6.2.9200.0 (x64)
[info]: OmniSharp.Services.DotNetCliService
        DotNetPath set to dotnet
[info]: OmniSharp.MSBuild.Discovery.MSBuildLocator
        Located 2 MSBuild instance(s)
            1: Visual Studio Professional 2019 16.7.30406.217 - "C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\MSBuild\Current\Bin"
            2: StandAlone 16.8.0 - "c:\Users\xarkam\\.vscode\extensions\ms-dotnettools.csharp-1.23.0\.omnisharp\1.36.1\.msbuild\Current\Bin"
[info]: OmniSharp.MSBuild.Discovery.MSBuildLocator
        Registered MSBuild instance: Visual Studio Professional 2019 16.7.30406.217 - "C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\MSBuild\Current\Bin"
[info]: OmniSharp.Cake.CakeProjectSystem
        Detecting Cake files in 'e:\Developpement\Base'.
[info]: OmniSharp.Cake.CakeProjectSystem
        Could not find any Cake files
[info]: OmniSharp.MSBuild.ProjectSystem
        Detecting projects in 'e:\Developpement\Base\SoftinuxBase.sln'.
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\SoftinuxBase.Infrastructure\SoftinuxBase.Infrastructure.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\SoftinuxBase.Barebone\SoftinuxBase.Barebone.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\SoftinuxBase.Security\SoftinuxBase.Security.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.Abstractions\SoftinuxBase.Security.Data.Abstractions.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.Entities\SoftinuxBase.Security.Data.Entities.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.EntityFramework.Sqlite\SoftinuxBase.Security.Data.EntityFramework.Sqlite.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.EntityFramework\SoftinuxBase.Security.Data.EntityFramework.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\SoftinuxBase.SeedDatabase\SoftinuxBase.SeedDatabase.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\Globalisation\Globalisation.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\SoftinuxBase.Data.Sqlite\SoftinuxBase.Data.Sqlite.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\SoftinuxBase.WebApplication\SoftinuxBase.WebApplication.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\WebApplication\WebApplication.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.EntityFramework.Mssql\SoftinuxBase.Security.Data.EntityFramework.Mssql.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.EntityFramework.Mysql\SoftinuxBase.Security.Data.EntityFramework.Mysql.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.EntityFramework.Npgsql\SoftinuxBase.Security.Data.EntityFramework.Npgsql.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\SoftinuxBase.Security.AuthorizeSetup\SoftinuxBase.Security.AuthorizeSetup.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\SoftinuxBase.Security.FeatureAuthorize\SoftinuxBase.Security.FeatureAuthorize.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\SoftinuxBase.Security.Permissions\SoftinuxBase.Security.Permissions.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\SoftinuxBase.Security.RefreshClaims\SoftinuxBase.Security.RefreshClaims.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\SoftinuxBase.Security.UserImpersonation\SoftinuxBase.Security.UserImpersonation.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\Testing\Unit\SoftinuxBase.SecurityTests\SoftinuxBase.SecurityTests.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\Testing\Unit\SoftinuxBase.Security.PermissionsTests\SoftinuxBase.Security.PermissionsTests.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\Testing\Extensions\SampleExtension1\SampleExtension1.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\Testing\Extensions\SampleExtension2\SampleExtension2.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\Testing\Unit\SoftinuxBase.Tests.Common\SoftinuxBase.Tests.Common.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for 'e:\Developpement\Base\src\Testing\Extensions\SampleExtension3\SampleExtension3.csproj'
[info]: OmniSharp.Script.ScriptProjectSystem
        Detecting CSX files in 'e:\Developpement\Base'.
[info]: OmniSharp.Script.ScriptProjectSystem
        Could not find any CSX files
[info]: OmniSharp.WorkspaceInitializer
        Invoking Workspace Options Provider: OmniSharp.Roslyn.CSharp.Services.CSharpFormattingWorkspaceOptionsProvider, Order: 0
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\SoftinuxBase.Infrastructure\SoftinuxBase.Infrastructure.csproj
[info]: OmniSharp.WorkspaceInitializer
        Invoking Workspace Options Provider: OmniSharp.Roslyn.CSharp.Services.RenameWorkspaceOptionsProvider, Order: 100
[info]: OmniSharp.WorkspaceInitializer
        Invoking Workspace Options Provider: OmniSharp.Roslyn.CSharp.Services.ImplementTypeWorkspaceOptionsProvider, Order: 110
[info]: OmniSharp.WorkspaceInitializer
        Invoking Workspace Options Provider: OmniSharp.Roslyn.CSharp.Services.BlockStructureWorkspaceOptionsProvider, Order: 140
[info]: OmniSharp.WorkspaceInitializer
        Configuration finished.
[info]: OmniSharp.Stdio.Host
        Omnisharp server running using Stdio at location 'e:\Developpement\Base' on host 18496.
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\SoftinuxBase.Infrastructure\SoftinuxBase.Infrastructure.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\SoftinuxBase.Infrastructure\SoftinuxBase.Infrastructure.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\SoftinuxBase.Barebone\SoftinuxBase.Barebone.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\SoftinuxBase.Barebone\SoftinuxBase.Barebone.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\SoftinuxBase.Barebone\SoftinuxBase.Barebone.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\SoftinuxBase.Security\SoftinuxBase.Security.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\SoftinuxBase.Security\SoftinuxBase.Security.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\SoftinuxBase.Security\SoftinuxBase.Security.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\SoftinuxBase.Security.Data.Abstractions\SoftinuxBase.Security.Data.Abstractions.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.Abstractions\SoftinuxBase.Security.Data.Abstractions.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.Abstractions\SoftinuxBase.Security.Data.Abstractions.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\SoftinuxBase.Security.Data.Entities\SoftinuxBase.Security.Data.Entities.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.Entities\SoftinuxBase.Security.Data.Entities.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.Entities\SoftinuxBase.Security.Data.Entities.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\SoftinuxBase.Security.Data.EntityFramework.Sqlite\SoftinuxBase.Security.Data.EntityFramework.Sqlite.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.EntityFramework.Sqlite\SoftinuxBase.Security.Data.EntityFramework.Sqlite.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.EntityFramework.Sqlite\SoftinuxBase.Security.Data.EntityFramework.Sqlite.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\SoftinuxBase.Security.Data.EntityFramework\SoftinuxBase.Security.Data.EntityFramework.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.EntityFramework\SoftinuxBase.Security.Data.EntityFramework.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.EntityFramework\SoftinuxBase.Security.Data.EntityFramework.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\SoftinuxBase.SeedDatabase\SoftinuxBase.SeedDatabase.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\SoftinuxBase.SeedDatabase\SoftinuxBase.SeedDatabase.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\SoftinuxBase.SeedDatabase\SoftinuxBase.SeedDatabase.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\Globalisation\Globalisation.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\Globalisation\Globalisation.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\Globalisation\Globalisation.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\SoftinuxBase.Data.Sqlite\SoftinuxBase.Data.Sqlite.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\SoftinuxBase.Data.Sqlite\SoftinuxBase.Data.Sqlite.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\SoftinuxBase.Data.Sqlite\SoftinuxBase.Data.Sqlite.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\SoftinuxBase.WebApplication\SoftinuxBase.WebApplication.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\SoftinuxBase.WebApplication\SoftinuxBase.WebApplication.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\SoftinuxBase.WebApplication\SoftinuxBase.WebApplication.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\WebApplication\WebApplication.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\WebApplication\WebApplication.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\WebApplication\WebApplication.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\SoftinuxBase.Security.Data.EntityFramework.Mssql\SoftinuxBase.Security.Data.EntityFramework.Mssql.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.EntityFramework.Mssql\SoftinuxBase.Security.Data.EntityFramework.Mssql.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.EntityFramework.Mssql\SoftinuxBase.Security.Data.EntityFramework.Mssql.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\SoftinuxBase.Security.Data.EntityFramework.Mysql\SoftinuxBase.Security.Data.EntityFramework.Mysql.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.EntityFramework.Mysql\SoftinuxBase.Security.Data.EntityFramework.Mysql.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.EntityFramework.Mysql\SoftinuxBase.Security.Data.EntityFramework.Mysql.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\SoftinuxBase.Security.Data.EntityFramework.Npgsql\SoftinuxBase.Security.Data.EntityFramework.Npgsql.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.EntityFramework.Npgsql\SoftinuxBase.Security.Data.EntityFramework.Npgsql.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\SoftinuxBase.Security.Data.EntityFramework.Npgsql\SoftinuxBase.Security.Data.EntityFramework.Npgsql.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\SoftinuxBase.Security.AuthorizeSetup\SoftinuxBase.Security.AuthorizeSetup.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\SoftinuxBase.Security.AuthorizeSetup\SoftinuxBase.Security.AuthorizeSetup.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\SoftinuxBase.Security.AuthorizeSetup\SoftinuxBase.Security.AuthorizeSetup.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\SoftinuxBase.Security.FeatureAuthorize\SoftinuxBase.Security.FeatureAuthorize.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\SoftinuxBase.Security.FeatureAuthorize\SoftinuxBase.Security.FeatureAuthorize.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\SoftinuxBase.Security.FeatureAuthorize\SoftinuxBase.Security.FeatureAuthorize.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\SoftinuxBase.Security.Permissions\SoftinuxBase.Security.Permissions.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\SoftinuxBase.Security.Permissions\SoftinuxBase.Security.Permissions.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\SoftinuxBase.Security.Permissions\SoftinuxBase.Security.Permissions.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\SoftinuxBase.Security.RefreshClaims\SoftinuxBase.Security.RefreshClaims.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\SoftinuxBase.Security.RefreshClaims\SoftinuxBase.Security.RefreshClaims.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\SoftinuxBase.Security.RefreshClaims\SoftinuxBase.Security.RefreshClaims.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\SoftinuxBase.Security.UserImpersonation\SoftinuxBase.Security.UserImpersonation.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\SoftinuxBase.Security.UserImpersonation\SoftinuxBase.Security.UserImpersonation.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\SoftinuxBase.Security.UserImpersonation\SoftinuxBase.Security.UserImpersonation.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\Testing\Unit\SoftinuxBase.SecurityTests\SoftinuxBase.SecurityTests.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\Testing\Unit\SoftinuxBase.SecurityTests\SoftinuxBase.SecurityTests.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\Testing\Unit\SoftinuxBase.SecurityTests\SoftinuxBase.SecurityTests.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\Testing\Unit\SoftinuxBase.Security.PermissionsTests\SoftinuxBase.Security.PermissionsTests.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\Testing\Unit\SoftinuxBase.Security.PermissionsTests\SoftinuxBase.Security.PermissionsTests.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\Testing\Unit\SoftinuxBase.Security.PermissionsTests\SoftinuxBase.Security.PermissionsTests.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\Testing\Extensions\SampleExtension1\SampleExtension1.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\Testing\Extensions\SampleExtension1\SampleExtension1.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\Testing\Extensions\SampleExtension1\SampleExtension1.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\Testing\Extensions\SampleExtension2\SampleExtension2.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\Testing\Extensions\SampleExtension2\SampleExtension2.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\Testing\Extensions\SampleExtension2\SampleExtension2.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\Testing\Unit\SoftinuxBase.Tests.Common\SoftinuxBase.Tests.Common.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\Testing\Unit\SoftinuxBase.Tests.Common\SoftinuxBase.Tests.Common.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\Testing\Unit\SoftinuxBase.Tests.Common\SoftinuxBase.Tests.Common.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: e:\Developpement\Base\src\Testing\Extensions\SampleExtension3\SampleExtension3.csproj
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file 'e:\Developpement\Base\src\Testing\Extensions\SampleExtension3\SampleExtension3.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project 'e:\Developpement\Base\src\Testing\Extensions\SampleExtension3\SampleExtension3.csproj'
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.Infrastructure
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.Barebone
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.Security
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.Security.Data.Abstractions
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.Security.Data.Entities
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.Security.Data.EntityFramework.Sqlite
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.Security.Data.EntityFramework
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.SeedDatabase
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: Globalisation
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.Data.Sqlite
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.WebApplication
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: WebApplication
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.Security.Data.EntityFramework.Mssql
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.Security.Data.EntityFramework.Mysql
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.Security.Data.EntityFramework.Npgsql
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.Security.AuthorizeSetup
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.Security.FeatureAuthorize
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.Security.Permissions
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.Security.RefreshClaims
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.Security.UserImpersonation
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.SecurityTests
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.Security.PermissionsTests
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SampleExtension1
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SampleExtension2
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SoftinuxBase.Tests.Common
[info]: OmniSharp.MSBuild.ProjectManager
        Update project: SampleExtension3
</details>

**Comment by broder-peters-k24** [comment]
Having the same errors on Arch Linux.

**Comment by filipw** [comment]
This is resolved here already https://github.com/OmniSharp/omnisharp-vscode/pull/3998
and will be part of the next (pre)release

**Comment by filipw** [comment]
This was released in 1.23.1
